packages/calling/src/Contacts/ContactsClient.ts

```diff
@@ -31,7 +31,18 @@ import {
 import {serviceErrorCodeHandler} from '../common/Utils';
 
 /**
+ * ContactsClient class represents the ContactsClient Module.
  * Client of contacts-service which stores encrypted custom contacts and org contacts.
+ * Purpose of this class is to provide the APIs to fetch and update the contacts and groups from contacts-service.
+ *
+ * To access the APIs, instance of ContactsClient is required.
+ *
+ * Example
+ * ```javascript
+ * const contactClient = createContactsClient(webex, logger);
+ * ```
+ *
+ * @implements {IContacts}
```

Comment:
Once again, we need not mention it as a class, rather address it as instance itself

Reply:
taken care